streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordCollector.java

```diff
@@ -19,13 +19,18 @@
 import org.apache.kafka.clients.producer.Producer;
 import org.apache.kafka.common.TopicPartition;
 import org.apache.kafka.common.header.Headers;
+import org.apache.kafka.common.serialization.ByteArraySerializer;
+import org.apache.kafka.common.serialization.BytesSerializer;
 import org.apache.kafka.common.serialization.Serializer;
 import org.apache.kafka.streams.processor.StreamPartitioner;
 
 import java.util.Map;
 
 public interface RecordCollector {
 
+    BytesSerializer BYTES_KEY_SERIALIZER = new BytesSerializer();
+    ByteArraySerializer BYTE_ARRAY_VALUE_SERIALIZER = new ByteArraySerializer();
+
```

Comment:
It doesn't seem like these need to be defined here, since they're only used outside of this interface.

They actually only have two, independent, usages, and it doesn't seem that important to de-duplicate the instances. Can we just copy them to separate constants in the classes that need them?

Reply:
Yeah, this was part of a larger refactoring that I walked back. That sounds like a reasonable request 👍